### ISSUE TITLE
Human friendly name in desktop entry

### ISSUE THREAD

**Issue by CardboardTurkey** [saivert/pwvucontrol#27]
The [desktop entry](https://github.com/saivert/pwvucontrol/blob/main/data/com.saivert.pwvucontrol.desktop.in.in#L2) gives this program the name `pwvucontrol`. However, when searching an app launcher (for example) it would be nicer to have a more descriptive human readable name like "PipeWire Volume Control". `pavucontrol` does something similar.

If you agree I would be happy to submit a PR for this but I'm not sure if it requires also changing things in [po](https://github.com/saivert/pwvucontrol/tree/main/po)

**Comment by saivert** [comment]
Yes you need to make it translatable as well and then run update the po files as [descibed](https://mesonbuild.com/Localisation.html).

You can then submit a PR with the updated `com.saivert.pwvucontrol.desktop` file and the update translation files.

**Comment by saivert** [comment]
Will be added in version 0.4.8.
